CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 -   **Breaking Change**: Rename `parity` to `geth` in the configuration file as we are only testing the software with Geth as part of the CI.
+-   **Breaking Change**: Depricate rfc003 routes by adding `rfc003` to GET and POST paths.
```

Comment:
```suggestion
-   **Breaking Change**: Deprecate rfc003 routes by adding `rfc003` to GET and POST paths.
```

Reply:
Wait, what is happening here? The paths have been designed from the beginning to be purely additive? Why did we have to make these changes?

Reply:
Let me check if I actually added this correctly to the changelog 😬

Reply:
My bad, there is not breaking change here. I removed the line.